### PR TITLE
ER-470 reporting fixes

### PR DESCRIPTION
**Pull request by chrisjensenuk**
Some fixes for issue raised during QA.

- Ensuring reporting parameters are UTC but displayed using local time
- Making the report header look more like v1 reports.

### CODE REVIEW

File: src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs [hunk 1]
```diff
@@ -49,23 +50,23 @@ public Task<Guid> PostCreateViewModelAsync(ProviderApplicationsReportCreateEditM
             switch (model.DateRange)
             {
                 case DateRangeType.Last7Days:
-                    fromDate = toDateInclusive.Date.AddDays(-7);
+                    fromDate = _timeProvider.Today.AddDays(-7);
```
Comment (shomavg) [inline src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs]: Can _timeProvider.Today be put into a variable and used in the other 3 locations?

File: src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs [hunk 2]
```diff
@@ -49,23 +50,23 @@ public Task<Guid> PostCreateViewModelAsync(ProviderApplicationsReportCreateEditM
...
                     break;
                 case DateRangeType.Last30Days:
-                    fromDate = toDateInclusive.Date.AddDays(-30);
+                    fromDate = _timeProvider.Today.AddDays(-30);
                     break;
                 case DateRangeType.Custom:
-                    fromDate = model.FromDate.AsDateTimeUk().Value.ToUniversalTime();
-                    toDateInclusive = model.ToDate.AsDateTimeUk().Value.ToUniversalTime().AddDays(1).AddTicks(-1);
+                    fromDate = model.FromDate.AsDateTimeUk(DateTimeStyles.AssumeLocal).Value.ToUniversalTime();
+                    toDateInclusive = model.ToDate.AsDateTimeUk(DateTimeStyles.AssumeLocal).Value.ToUniversalTime().AddDays(1).AddTicks(-1);
```
Comment (shomavg) [inline src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs]: I'm just a bit confused here! What is happening here? Will having some comments help?

File: src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs (hunk 1 above)
Comment (chrisjensenuk) [reply src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs]: I'm not sure adding another variable into the mix simplifies this any. I think its descriptive enough.

Comment (sachpatel) [reply src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs]: seems fine to me

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/CsvBuilder.cs [hunk 3]
```diff
@@ -45,23 +45,49 @@ private static CsvWriter GetCsvWriter(TextWriter streamWriter)
 
         private void WriteNoResults(CsvWriter csv, DateTime reportDate)
         {
+            WriteTotalHeader(csv, reportDate, 0, 2);
+        }
+
+        private void WriteTotalHeader(CsvWriter csv, DateTime reportDate, int total, int fieldCount)
+        {
+            csv.WriteField("PROTECT");
```
Comment (sachpatel) [inline src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/CsvBuilder.cs]: What does PROTECT do?

File: src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs (hunk 1 above)
Comment (shomavg) [reply src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs]: 👍

File: src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/CsvBuilder.cs (hunk 3 above)
Comment (chrisjensenuk) [reply src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/CsvBuilder.cs]: Nothing its just text.  Its on the v1 reports. I think its a data protection hint.

**Pull request merged**